webui/react/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getTensorBoards({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+      ]);
+      const newTasks = [...commands, ...jupyterLabs, ...shells, ...tensorboards];
+      setTasks(newTasks);
+    },
+    [canceler],
+  );
+
+  const fetchExperiments = useCallback(async (user: DetailedUser) => {
+    const response = await getExperiments({
+      orderBy: 'ORDER_BY_DESC',
+      sortBy: 'SORT_BY_START_TIME',
+      userIds: [user.id],
+    });
+    setExperiments(response.experiments);
+  }, []);
+
+  const fetchProjects = useCallback(async () => {
+    const projects = await getProjectsByUserActivity({ limit: 12 });
+    setProjects(projects);
+  }, []);
+
+  useEffect(() => {
+    fetchProjects();
+    if (!authUser) return;
+    fetchExperiments(authUser);
+    fetchTasks(authUser);
+  }, [authUser, fetchExperiments, fetchTasks, fetchProjects]);
+
+  useEffect(() => {
+    let submissions: Submission[] = [];
+    submissions = submissions
+      .concat(experiments as Submission[])
+      .concat(tasks as Submission[])
+      .sort((a, b) => dateTimeStringSorter(b.startTime, a.startTime));
+    setSubmissions(submissions);
+  }, [experiments, tasks]);
+
+  return (
+    <Page title="Home">
+      <Section title="Recent projects">
+        <Grid
+          count={projects.length}
+          gap={ShirtSize.Medium}
+          minItemWidth={250}
+          mode={GridMode.ScrollableRow}>
+          {projects.map((project) => (
+            <ProjectCard
+              curUser={authUser}
+              fetchProjects={fetchProjects}
+              key={project.id}
+              project={project}
+            />
+          ))}
+        </Grid>
+      </Section>
+      <Section title="Recently submitted">
+        <Table<Submission>
+          columns={[
+            {
+              dataIndex: 'state',
+              render: (state) => {
+                return <ExperimentIcons state={state} />;
+              },
+              width: 1,
+            },
+            {
+              dataIndex: 'projectId',
+              render: (projectId) => {
+                if (projectId) {
+                  return <Icon name="experiment" />;
+                } else {
+                  return <Icon name="tasks" />;
+                }
+              },
+              width: 1,
+            },
+            {
+              dataIndex: 'name',
+              render: (name, row) => {
+                if (row.projectId) {
+                  // only for Experiments, not Tasks:
+                  return experimentNameRenderer(name, row);
+                } else {
+                  return name;
+                }
+              },
+            },
+            {
+              dataIndex: 'projectId',
```

Comment:
project name and prob workspace name should be hyper linked like name.

<img width="961" alt="image" src="https://user-images.githubusercontent.com/109113616/207957867-ff5f827f-944f-4bad-926b-d10a50249df4.png">

Reply:
project name already is a link. adding link to workspace name.

Reply:
thanks. also text color should be blue?

Reply:
i think the link colors are all in line with the figma and what we do on other tables